tests/integration/security/sds_vault_flow/sds_vault_flow_test.go

```diff
@@ -85,16 +85,17 @@ func TestSdsVaultCaFlow(t *testing.T) {
 				},
 			}
 
-			// Apply the policy to the system namespace.
-			deployment := tmpl.EvaluateOrFail(t, file.AsStringOrFail(t, "testdata/global-mtls.yaml"),
+			// Apply the policy
+			deployment := tmpl.EvaluateOrFail(t, file.AsStringOrFail(t, "testdata/config.yaml"),
 				map[string]string{
 					"Namespace": ns.Name(),
 				})
-			g.ApplyConfigOrFail(t, systemNS, deployment)
-			defer g.DeleteConfigOrFail(t, systemNS, deployment)
 
-			// Sleep 3 seconds for the policy to take effect.
-			time.Sleep(3 * time.Second)
+			g.ApplyConfigOrFail(t, ns, deployment)
+			defer g.DeleteConfigOrFail(t, ns, deployment)
+
+			// Sleep 10 seconds for the policy to take effect.
+			time.Sleep(10 * time.Second)
```

Comment:
same here why 10 seconds?

Reply:
3 seconds may not be enough for a policy to take effect. 10 seconds gives more time for a policy to take effect.